notebooks/official/training/sdk2_remote_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,1771 @@
+{
```

Comment:
<ul><li>It would be better to make it clear about "Supported", for example, "Remote tuning supported ML frameworks:" and "Local tuning supported ML frameworks:"</li><li>Suggest to capitalize the items on the list when appropriate, e.g. "Custom model", "Pytorch Lightning", etc.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>